amaranth/vendor/xilinx.py

```diff
@@ -949,7 +1056,7 @@ def get_diff_input(self, pin, port, attrs, invert):
         return m
 
     def get_diff_output(self, pin, port, attrs, invert):
-        if self.toolchain == "Symbiflow":
+        if self.toolchain == ["Symbiflow", "yosys_nextpnr"]:
```

Comment:
this looks like obvious mistake

Reply:
Fixed. I've replaced this test by the use of `self.oss_toolchain` as done for rest of methods